queries/groovy/highlights.scm

```diff
@@ -20,10 +20,18 @@
 ((identifier) @constant
   (#lua-match? @constant "^[A-Z][A-Z%d_]*$"))
 
+((identifier) @constant.builtin
+ (#eq? @constant.builtin "null"))
+
 ((identifier) @type.definition
   (#any-of? @type.definition
     "string"
     "String"
+    "Map"
+    "Object"
+    "Boolean"
+    "Integer"
+    "List"
     "void"
```

Comment:
I just noticed this shouldn't be `@type.definition`.
I'd make these `@type` and the rest `@function.builtin`.

Reply:
Changed